tests/phpunit/elementor/modules/campaigns/test-module.php

```diff
@@ -0,0 +1,62 @@
+<?php
+namespace Elementor\Testing\Modules\Campaigns;
+
+use Elementor\Plugin;
+use Elementor\Testing\Elementor_Test_Base;
+use Elementor\Modules\Campaigns\Module;
+use Elementor\Tests\Phpunit\Test_Upgrades_Trait;
+
+class Elementor_Test_Campaigns_Module extends Elementor_Test_Base {
+
+	use Test_Upgrades_Trait;
+
+	/** @var Module */
+	private $module;
+
+	public function setUp() {
+		parent::setUp();
+
+		$this->module = Plugin::$instance->modules_manager->get_modules( 'campaigns' );
+	}
+
+	public function test___construct() {
+		// Assert that the taxonomy has been created.
+		$this->assertTrue( taxonomy_exists( Module::TAXONOMY_NAME ) );
+
+		// Assert that the $campaigns_menu_slug class variable was set.
+		// A separate test (test_get_menu_slug()) determines if it was set correctly.
+		$this->assertNotNull( $this->module->get_menu_slug() );
+	}
+
+	public function test_get_name() {
+		$expected_module_name = 'campaign';
+		$actual_module_name = $this->module->get_name();
+
+		$this->assertEquals( $expected_module_name, $actual_module_name );
+	}
+
+	public function test_get_taxonomy_page_slug() {
+		$expected_slug = 'edit-tags.php?taxonomy=' . Module::TAXONOMY_NAME . '&post_type=page';
+		$actual_slug = $this->module->get_taxonomy_page_slug();
+
+		$this->assertEquals( $expected_slug, $actual_slug );
+	}
+
+	public function test_get_taxonomy_page_slug__without_page_filter() {
+		$expected_slug = 'edit-tags.php?taxonomy=' . Module::TAXONOMY_NAME;
+		$actual_slug = $this->module->get_taxonomy_page_slug( false );
+
+		$this->assertEquals( $expected_slug, $actual_slug );
+	}
+
+	// This method has to run before `test_get_menu_slug__with_terms()` because `test_get_menu_slug__with_terms()`
```

Comment:
In order to avoid this problem, implement the `tearDown()` method and clear the terms

Reply:
This will only happen once we have a solution for testing the constructor. Removing both the comment and the TODO.